README.md

```diff
@@ -27,8 +30,78 @@ config = {
     'region': 'us-east-2',
     'subscriber': '00000000000',       
     'name': 'profile',
-    'env': 'prod'
+    'env': 'dev'
+}
+
+publisher = get_publisher(config=config, pub_type='sns')
+print(publisher.arn)
+>>> 'arn:aws:sns:us-east-2:00000000000:profile--dev'
+```
+
+Publish:
+
+```python
+from elife_bus_sdk.events import Event
+
+publisher.publish(Event(foo='bar', some='fields'))
+
+```
+
+Queue
+---------
+
+Current supported:
+- [SQS](https://aws.amazon.com/sqs/)
+
+
+Configuration:
+
+```python
+from elife_bus_sdk import get_queue
+
+config = {   
+    'QueueName': 'some_queue',
```

Comment:
this should be `queue_name` for consistency with the other configurations for this library?

Reply:
It's should be 'snake_case' yes, but `aws` make their own rules. It's how they require it, I am just passing through the values and not renaming everything.

Reply:
In the SNS publisher module, this project is trying to hide that there is an AWS service behind the interface, and it seems the queue module is trying to do the same. The queue name should normally be the only configuration that comes from outside (timeouts or message visibility are rarely tuned), so a standard name would complete the job.

Reply:
Ok, will change and handle the conversion.